### ISSUE TITLE
Add Hide Self-Repost/Retweet

### ISSUE THREAD

**Issue by Brandin** [insin/control-panel-for-twitter#452]
Hi there,

I've been looking for a feature to disable self-repost/re-tweets from my timeline. I do like seeing some reposts and retweets from others, but I definitely don't want someone to spam my timeline by continuing to repost and retweet their own tweets. Does this feature exist in this extension and I missed it?

Thanks!
Brandin.

**Comment by sashazykov** [comment]
I am here to create the same request.

**Comment by conoro** [comment]
Yup, just came here to ask the same thing please. The thread-jockeys seem to retweet themselves all day long.

**Comment by Brandin** [comment]
Did some work on this over in my fork if someone wants to test it. No setting in UI yet, testing via UserScript currently. Once I'm comfortable it's working and the code addition isn't heavy I'll promote a PR. 

https://github.com/Brandin/control-panel-for-twitter/blob/master/script.js
